src/test/scala/com/amazon/deequ/analyzers/runners/AnalysisRunnerTests.scala

```diff
@@ -341,5 +341,23 @@ class AnalysisRunnerTests extends WordSpec with Matchers with SparkContextSpec w
         inputStream => assert(inputStream.read() > 0)
       }
     }
+
+    "should give error for duplicate analyzers" in withSparkSession { sparkSession =>
+
+      val df = getDfWithNumericValues(sparkSession)
+
+      val analyzers = Size() :: Completeness("item") :: Size() :: Nil
+
+      try {
```

Comment:
I recommend to use ScalaTest's `assertThrows` instead of `try / catch` block, like:
```
assertThrows[IllegalArgumentException] { 
     AnalysisRunner.onData(df)
          .addAnalyzers(analyzers)
          .useSparkSession(sparkSession)
          .run()
}
```

See details here: http://www.scalatest.org/user_guide/using_assertions

Reply:
Thanks for the suggestion, I have made changes and used intercept for it, does that work ? With assertThrows there is version conflict, not able to find

Reply:
ok, looks good